modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -383,6 +386,244 @@ bool Visualizer::reset_key() {
   return true;
 }
 
+double Visualizer::regularize_angle(const double &radian_angle) {
```

Comment:
Simply pass value of primitive types, instead of reference. Plz also fix others.

Reply:
I think reference consumes less memory than copying double type.

Reply:
It takes as long as a pointer, so generally there is no benefit. Mordern compiler could use register to store the scala types which even gives better performance.

Reply:
OK. I will change double reference to call by value.

Reply:
I updated the code to have to call by value of double type!